logdna/resource_stream_exclusion.go

```diff
@@ -161,7 +161,8 @@ func resourceStreamExclusion() *schema.Resource {
 			},
 			"title": {
 				Type:     schema.TypeString,
-				Required: true,
+				Default:  "",
```

Comment:
oof, sorry I'm seeing this other change late: https://github.com/answerbook/logdna-api/pull/435#discussion_r699605588

TBH, I would have voted against this.  I see your thought process for "well, it's a string", but IMO it should be `null`.  That should be a simple change to the Joi model to allow both.  It depends on the consumers too, but we desperately need better JSON standards, and using `null` for empty values is the standard.  What's our ability to change it?

Reply:
Also, mongo has special treatment for null, so using `''` has query implications as well.

Reply:
@darinspivey ah gotcha, yeah that makes sense to me. My only concern would be restoring the inconsistency between API and Web but this doesn't really cause any "new" issues since that's already how it was before

I'm good with updating the Joi schema to allow `null` and set it as the default value of the `title` property but we probably wanna loop in product since this specific change was an IBM ask

Reply:
Yep, I hear ya, we gotta be careful, but I do think we have to try and do better where we can.  Those types are things are the old patterns that make our data non-standard.  If we can change it now, let's do it.

Reply:
@david-ly do you have an update if we'll get to address this issue?  I'm fine approving this if we're going to address it.  We spent LOTS of time cleaning up this provider, and lots of the issues were related to bad values in the APIs such as string booleans ("false").  This empty string feels similar when it should be `null`.

Reply:
@darinspivey yeah I just put out a PR in the logdna-api repo to facilitate this https://github.com/answerbook/logdna-api/pull/456. As far as the provider goes, I don't think there's any major updates aside from aligning the property with API by setting its default value to `nil` and making it optional. This commit makes this change alongside some minor housecleaning items.